tests/issues/issue-1917.php

```diff
@@ -130,4 +130,47 @@
         'standard' => true,
         'full' => true,
     ],
+    'issue-1917-D' => [
+        'ua' => 'Mozilla/5.0 (Linux; Android 6.0.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.113 Whale/0.9.5.0 Mobile Safari/537.36',
+        'properties' => [
+            'Comment' => 'Whale Browser',
+            'Browser' => 'Whale Browser',
+            'Browser_Type' => 'Browser',
+            'Browser_Bits' => '32',
+            'Browser_Maker' => 'NHN Corporation',
+            'Browser_Modus' => 'unknown',
+            'Version' => '0.9',
```

Comment:
Please change the version to `0.0`. This let the test fail.

Reply:
Changed here to `0.0`. Thank you for your generosity.